src/utilities/plugin-utils.ts

```diff
@@ -26,7 +26,7 @@ const addUsedPlugin = (plugin: Plugin) => {
 
 export const findUsedPlugins = (activity: Activity, teacherEditionMode: boolean) => {
   // search each page for teacher edition plugin use
-  for (let page = 0; page < activity.pages.length - 1; page++) {
+  for (let page = 0; page <= activity.pages.length - 1; page++) {
```

Comment:
```suggestion
  for (let page = 0; page < activity.pages.length; page++) {
```
Seems like the fix could be (slightly) simpler.

Reply:
Fixed